.github/workflows/docker-image.yml

```diff
@@ -265,7 +312,46 @@ jobs:
 
       - name: ensure systemd container is gone
         run: sudo podman ps -a
-        
+
+  systemd-enable-validate:
+    runs-on: ubuntu-latest
+    needs: [ build ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Enable the podman socket
+        run: sudo systemctl enable --now podman.socket
+
+      - name: pull artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: harpoon-image
+          path: /tmp
+
+      - name: Load the image
+        run: sudo podman load -i /tmp/harpoon.tar
+
+      - name: tag the image
+        run: sudo podman tag quay.io/harpoon/harpoon-amd:latest quay.io/harpoon/harpoon:latest
+
```

Comment:
we will need to do the same steps of loading the systemd image and tagging it as harpoon-systemd:latest here that way we test against the newly created image

Reply:
that should allow for us to pass the failing test